### ISSUE TITLE
cache settings not applying to endpoints in v1.2.x

### ISSUE THREAD

**Issue by willfarrell** [DianaIonita/serverless-api-gateway-caching#30]
Tested:
- v1.2.2
- v1.2.1
- v1.2.0
- v1.1.6 <-- Works

**serverless.yml**
```yml
service: template

custom:
  cors:
    origin: '*'
    headers:
      - Authorization
      - Content-Type
      - X-Amz-Date
      - X-Amz-Security-Token
      - X-Amz-User-Agent
      - X-Api-Key
    allowCredentials: false
  contentEncoding:
    contentCompression: 1400
  apiGatewayCaching:
    enabled: true
    clusterSize: '0.5'
    ttlInSeconds: 300
    # TODO encrypt cache
    perKeyInvalidation:
      requireAuthorization: false
      handleUnauthorizedRequests: Ignore

provider:
  name: aws
  runtime: nodejs8.10
  versionFunctions: false
  region: "ca-central-1"

package:
  individually: false
  excludeDevDependencies: true
  exclude:
    - ./**
  include:
    - package.json
    - node_modules/**
    - '!node_modules/serverless/**'
    - '!node_modules/serverless-*/**'
    - src/**

functions:
  health:
    description: Used for health checks
    memorySize: 128
    timeout: 30
    handler: src/handlers/health.handler
    events:
      - http:
          path: health
          method: GET
          cors: ${self:custom.cors}
          caching:
            enabled: true
            ttlInSeconds: 3600

plugins:
  - serverless-api-gateway-caching
```

**Comment by wmegel** [comment]
check the 'method' field in your function definition.
You must define it in lowercase (I checked the plugin code because i had the same issue).

**Comment by DianaIonita** [comment]
Thanks @willfarrell and @wmegel - I've just released v1.3.0 which should fix this issue. Let me know if you still run into trouble.
